pkg/server/init.go

```diff
@@ -390,6 +390,24 @@ func (s *initServer) startJoinLoop(ctx context.Context, stopper *stop.Stopper) e
 		return ErrJoinRPCUnsupported
 	}
 
+	// Busy-loop through all the resolvers at least once. Keep this code block
```

Comment:
wording: This is not a busy-loop (a busy-loop loops back into itself). You can just say "Iterate through"